claw_controller:
  ros__parameters:
    # this is the correct pin on our rpi (BCM mode)
    in1_pin: 21 
    in2_pin: 20 
    en_pin: 6 
    min_claw_value: 0.0 # 0V
    max_claw_value: 5.0 # 5V

ball_grabber:
  ros__parameters:
    min_claw_value: 1.5 # TODO: adjut this vallue
    min_claw_value: 3.5 # TODO: adjut this vallue